rclcpp/include/rclcpp/node_options.hpp

```diff
@@ -372,7 +371,9 @@ class NodeOptions
     rclcpp::QoSInitialization::from_rmw(rmw_qos_profile_parameter_events)
   );
 
-  rclcpp::QoS rosout_qos_ = rclcpp::QoS(rcl_qos_profile_rosout_default);
+  rclcpp::QoS rosout_qos_ = rclcpp::RosoutQoS(
+    rclcpp::QoSInitialization::from_rmw(rcl_qos_profile_rosout_default)
```

Comment:
Isn't this redundant now? Since this is the same as the default argument of the `rclcpp::RosoutQoS` class constructor?

Reply:
Changed.